sql/core/src/test/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListenerSuite.scala

```diff
@@ -55,12 +55,10 @@ class SQLAppStatusListenerSuite extends SparkFunSuite with SharedSQLContext with
 
   private var kvstore: ElementTrackingStore = _
 
-  before {
-    kvstore = new ElementTrackingStore(new InMemoryStore, sparkContext.conf)
-  }
-
   after {
-    kvstore.close()
+    if (kvstore != null) {
+      kvstore.close()
```

Comment:
After calling close please release the store too: set kvstore to null.

Reply:
Yes. we should make it null. Thanks @attilapiros